scaffolder-skeletons/manifests-skeleton/manifests/base/service.yaml

```diff
@@ -1,12 +1,12 @@
 kind: Service
 apiVersion: v1
 metadata:
-  name: ${{ values.repo_name }}
+  name: ${{ values.repoName }}
```

Comment:
It's missing the labels in the service.yaml
labels:
    app: ${{ values.repoName }}
    app.kubernetes.io/component: backstage
    backstage.io/kubernetes-id: ${{ values.namespace }}